src/js/components/Select/SelectContainer.js

```diff
@@ -36,22 +40,60 @@ const SelectOption = styled(Button)`
   width: 100%;
 `;
 
-const ClearButton = ({ clear, onClear, name, theme, setFocus }) => {
+const ClearButtonContainer = styled(Box)``;
+const ClearButtonText = styled(Text)``;
+const ClearButtonComponent = ({
+  className,
+  clear,
+  onClear,
+  name,
+  theme,
+  setFocus,
+}) => {
   const { label, position } = clear;
   const align = position !== 'bottom' ? 'start' : 'center';
   const buttonLabel = label || `Clear ${name || 'selection'}`;
+
   return (
     <Button
+      // className must be passed for styled-components component interpolation
+      // https://styled-components.com/docs/advanced#referring-to-other-components
+      className={className}
       onClick={onClear}
       onFocus={() => setFocus(true)}
       onBlur={() => setFocus(false)}
     >
-      <Box {...theme.select.clear.container} align={align}>
-        <Text {...theme.select.clear.text}>{buttonLabel}</Text>
-      </Box>
+      <ClearButtonContainer {...theme.select.clear.container} align={align}>
```

Comment:
Do we need to create a separate styled component on L43 that is empty? Instead could we not just spread the theming directly on the Box?
```suggestion
      <Box {...theme.select.clear.container} align={align}>
```

Reply:
Or is it necessary for the below interpolation?

Reply:
Hey sorry for the late reply. It’s necessary for the interpolation. Shall I add a comment next to the component definitions to explain it?